macros/address.njk

```diff
@@ -10,51 +10,53 @@
 {% endmacro %}
 
 {% macro postcodeLookupAddressFragment(form, name) %}
-  {% set internalName = name + '[postcodeLookup]' if name else 'postcodeLookup' %}
```

Comment:
this is missing the changes from: https://github.com/hmcts/cmc-common-frontend/pull/79
which:
1. makes response address layout consistent with make claim
2. fixes response
3. makes int tests work by adding an id for enter manually